R/rvar-bind.R

```diff
@@ -69,7 +73,13 @@ bind_rvars <- function(args, arg_exprs, deparse.level = 1, axis = 1) {
 
 #' broadcast two rvars to compatible dimensions and bind along the `axis` dimension
 #' @noRd
-broadcast_and_bind_rvars <- function(x, y, axis = 1) {
+broadcast_and_bind_rvars <- function(x, y, axis = 1) UseMethod("broadcast_and_bind_rvars")
+
+#' @export
+broadcast_and_bind_rvars.rvar <- function(x, y, axis = 1) {
+  # TODO: should really replace this with something that takes a list of rvars,
```

Comment:
How urgent/important is that TODO? Is it worth including this already in the current PR?